fuel/src/test/kotlin/com/github/kittinunf/fuel/RoutingTest.kt

```diff
@@ -172,18 +175,16 @@ class RoutingTest: MockHttpTestCase() {
         val (data, error) = result
 
         // Binary data is encoded in base64 by mock server
-        val string = String(Base64.decode(JSONObject(data).getJSONObject("body").getString("base64Bytes"), Base64.DEFAULT))
+        val string = JSONObject(data).getJSONObject("body").getString("base64Bytes").decodeBase64()
```

Comment:
You might need to decode twice!!

The current `reflect()` template base64 encoded binary data! So if the input is a binary base64 encoded string, it will be encoded twice. That is the expected behaviour here, which is why this is pre-decoding the string and on line 185 it was actually decoding.